src/sentry/search/snuba/backend.py

```diff
@@ -49,6 +49,21 @@ def unassigned_filter(unassigned, projects):
     return query
 
 
+def first_release_all_environments_filter(version, projects):
+    try:
+        release_id = Release.objects.get(
+            organization=projects[0].organization_id, version=version
+        ).id
+    except Release.DoesNotExist:
+        release_id = -1
```

Comment:
This is kind of gross, I could potentially introduce an exception here that gets caught and just returns an empty list.

Reply:
would that be better without having to hit `GroupEnvironment`?

Reply:
It'd just let us skip making a query at all, maybe I'll follow up with that change later.